config/controller.yaml

```diff
@@ -65,6 +74,9 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
```

Comment:
This seems way to low. Can we default to 1 cpu and 1gb?

Reply:
Sure! I think these are pretty low as well, but didn't know if you had these for a reason. I want this to be able to fit on nodes with 1vCPU as well (assuming not that many daemonsets). WDYT about `800m` and `1Gi` ?

Reply:
Nodes with 1vcpu (the t3.micros) have much more overhead than that 800 won't fit. Feel free to test it out ;)

Reply:
For reference, `kube-proxy` requests `100m` and `aws-node` requests `10m`

Reply:
When I was doing some testing for the demo, I saw the Karpenter pod got OOM killed. IIRC, it was set to 20Mi limit. I think the real number can only be decided with scale testing. 
Can we skip setting limits for now, or do we need for something?

Reply:
I'd like to at least increase it to make it a little more resistant to OOM killing until we get the scale testing done.

Reply:
Agree with Nick.